### ISSUE TITLE
How can we send field in Delete Action which are not Key

### ISSUE THREAD

**Issue by praveenjain6407** [hikalkan/jtable#1745]
I need to send fields in delete action which are not tagged as Key. As In my Case I want to delete a row using Multiple fields.

**Comment by sajjadpz** [comment]
you can send fields as part of your HTTP request header
for example
/Controller?action=delete&field1=value&filed2=value

**Comment by praveenjain6407** [comment]
You are right ... But thing that We need to select value dynamically based on rows .. So How can we retrieve that values?

**Comment by sajjadpz** [comment]
you need to redefine deleteAction since the default delete behavior only supports deletion based on single key.
I have achieved it like this.
//Add custom delete button to jtable.
Delete: {
										    title: '',
										    create: false,
										    width: '0.3%',
										    display: function(data){
										        var $img = $('');
										        $img.click(function(){
										            $('#container').jtable('deleteRecord', {
										                key: data.record.region,
										                url:'../Controller?action=delete&field1='
															+ data.record.filed1name
															+ '&field2='
															+ data.record.filed2name
										            });
										        });
										        return $img;
										    }
										}
